From 97d8d821120596301a3885112c9c3cb21158259a Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <guoqing.jiang@suse.com>
Date: Thu, 1 Sep 2022 10:16:26 +0800
Subject: [PATCH] Revert "Linux 5.10.140"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No


This reverts commit 18ed766f3642fa75262885462d3052ad7c8c87a2.

Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a80179d2c005..48140575f960 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 140
+SUBLEVEL = 139
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

